docs/substitutions/global.rst

```diff
@@ -40,4 +40,8 @@
 
 .. |disable_notification| replace:: Sends the message silently. Users will receive a notification with no sound.
 
-.. |reply_to_msg_id| replace:: If the message is a reply, ID of the original message.
\ No newline at end of file
+.. |reply_to_msg_id| replace:: If the message is a reply, ID of the original message.
+
+.. |sequenceeclassargs| replace:: Accepts any :class:`collections.abc.Sequence` as input instead of just a list. The input is converted to a tuple.
```

Comment:
```suggestion
.. |sequenceclassargs| replace:: Accepts any :class:`collections.abc.Sequence` as input instead of just a list. The input is converted to a tuple.
```
almost there 😁

Reply:
🤦